spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -188,6 +199,24 @@ data class SpdxCompoundExpression(
         right.validate(strictness)
     }
 
+    override fun validChoicesForDnf(): Set<SpdxExpression> =
+        when (operator) {
+            SpdxOperator.AND -> setOf(this)
+            SpdxOperator.OR -> {
```

Comment:
Nit: Maybe a blank line before this one, even if the previous case is not a block?

Reply:
Done.